yml/microsoft/built-in/wptsextensions.yml

```diff
@@ -0,0 +1,15 @@
+---
+Name: wptsextensions.dll
+Author: k4nfr3
+Created: 2022-08-15
+Vendor: Microsoft
+VulnerableExecutables:
+  - Path: 'Task Scheduler'
+    Type: Environment Variable
+    PrivilegeEscalation: true
+    Variable: PATH
+Resources:
+  - "https://itm4n.github.io/windows-dll-hijacking-clarified/"

```

Comment:
Probably better to use the original resource instead: http://remoteawesomethoughts.blogspot.com/2019/05/windows-10-task-schedulerservice.html